src/raftstore/store/peer_storage.rs

```diff
@@ -1130,6 +1123,22 @@ impl PeerStorage {
     }
 }
 
+fn get_sync_log_from_entry(entry: &Entry) -> bool {
+    if entry.get_sync_log() {
+        return true;
```

Comment:
Why return?

Reply:
Hint?

Reply:
Usually using return means wanting to reduce indent. But here you also write the else branch. Usually write both if and else branch means using expression style, but here you use return. It's confusing.